Kitodo-API/src/main/java/org/kitodo/api/dataformat/TreeNode.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.dataformat;
+
+import java.util.List;
+
+public interface TreeNode<T> {
```

Comment:
I agree, that's a good idea. Especially because it should prevent any confusions with the `TreeNode` class from PrimeFaces, which we now use extensively.